api.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+const getArticles = () => {
+  return axios
+    .get("https://backend-nc-2024.onrender.com/api/articles")
+    .then(response => {
+      console.log(response.data.articles);
+      return response.data.articles;
+    })
+    .catch(error => {
+      console.log("Thats an error:", error);
+    });
+};
+
+const getArticle = () => {
+  return axios
+    .get("https://backend-nc-2024.onrender.com/api/articles/:article_id")
+    .then(response => {
+      console.log(response.data);
+      return response.data;
+    })
+    .catch(error => {
+      console.log(error);
+    });
+};
+
+export default (getArticles, getArticle);
```

Comment:
we cannot export 2 things as default. please change this and export only one thing as default. to export other things you will have to use the syntax 
```js
export const getArticle = () => {...code here}
```